tools/integration_tests_runner.py

```diff
@@ -58,13 +64,29 @@ def exec_tests_in_queue(self):
                     return
 
                 test_file, test = self.tests.get()
-                self.exec_test(test_file, test)
+
+#                    test_file = '{} "--user-resource-path={}"'.format(test_file, tmpdir) #--data_path
+
+                self.exec_test(test_file, test, clean_env=(test_file in TEMPFOLDER_TESTS))
+
             except:
                 logging.error(traceback.format_exc())
 
 
-    def exec_test(self, test_file, test):
-        out, err, result = self.get_tests_from_exec_file(test_file, '--filter={test}'.format(test=test))
+    def exec_test(self, test_file, test, clean_env=False):
+        keys = '--filter={test}'.format(test=test)
+        if clean_env:
+            tmpdir = tempfile.mkdtemp()
+            logging.debug("Temp dir: {}".format(tmpdir))
```

Comment:
Some of our systems has a old python interpreter. They can't handle {} string formatting. Please, use {0} instead.